13/umbraco-engage/developers/headless/README.md

```diff
@@ -6,14 +6,14 @@ description: >-
 
 # Headless
 
-Umbraco Engage has a **Umbraco.Engage.Headless** package that can be installed to integrate with Umbraco 12.0+. Headless Content Delivery API, enabling personalized content, A/B tests, and segmentation.
+Umbraco Engage offers the **Umbraco.Engage.Headless** package for seamless integration with Umbraco 12.0 and later. This package enables access to the Headless Content Delivery API, enabling personalized content, A/B tests, and segmentation.
 
 ## Requirements
 
-To install Umbraco.Engage.Headless, ensure the following:
+To install Umbraco.Engage.Headless, ensure the following prerequisites:
 
 * Umbraco v12 or higher is required to integrate with the [Content Delivery API](https://docs.umbraco.com/umbraco-cms/reference/content-delivery-api).
```

Comment:
Umbraco Engage is only for 13 currently, so let's either get rid of this line, or update it to 13 💪 😁

Reply:
That's a good point. I'll update it to v13.